### PR TITLE
reference/tools: update binlogctl doc to add "-show-offline-nodes"

### PR DESCRIPTION
**Pull request by suzaku**
### What is changed, added or deleted? 

With pingcap/tidb-binlog#774 merged, `binlogct` no longer shows offline pumps/drainers by default.
The related document is updated to reflected the change.

### What is the related PR or file link(s)? 

pingcap/tidb-binlog#774

### Which version does your change affect? 

- 2.1
- 3.0

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
LGTM

File: dev/reference/tools/tidb-binlog/maintain.md [hunk 1]
```diff
@@ -112,6 +112,9 @@ Usage of binlogctl:
         PEM 格式的 X509 key 文件的路径
 -time-zone string
     如果设置时区，在 "generate_meta" 模式下会打印出获取到的 tso 对应的时间。例如"Asia/Shanghai" 为 CST 时区，"Local" 为本地时区
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/maintain.md]:
```suggestion
    如果设置时区，在 "generate_meta" 模式下会打印出获取到的 tso 对应的时间。例如 "Asia/Shanghai" 为 CST 时区，"Local" 为本地时区
```

File: dev/reference/tools/tidb-binlog/maintain.md [hunk 2]
```diff
@@ -112,6 +112,9 @@ Usage of binlogctl:
...
+-show-offline-nodes
+    在用 -cmd pumps 或 -cmd drainers 时使用，这两个命令默认不显示 offline 的节点，仅当明确指定 -show-offline-nodes 时会显示
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/maintain.md]:
```suggestion
    在用 `-cmd pumps` 或 `-cmd drainers` 命令时使用，这两个命令默认不显示 offline 的节点，仅当明确指定 `-show-offline-nodes` 时会显示
```

File: v2.1/reference/tools/tidb-binlog/maintain.md [hunk 3]
```diff
@@ -112,6 +112,8 @@ Usage of binlogctl:
         PEM 格式的 X509 key 文件的路径
 -time-zone string
     如果设置时区，在 "generate_meta" 模式下会打印出获取到的 tso 对应的时间。例如"Asia/Shanghai" 为 CST 时区，"Local" 为本地时区
```
Comment (lilin90) [inline v2.1/reference/tools/tidb-binlog/maintain.md]:
```suggestion
    如果设置时区，在 "generate_meta" 模式下会打印出获取到的 tso 对应的时间。例如 "Asia/Shanghai" 为 CST 时区，"Local" 为本地时区
```

File: v2.1/reference/tools/tidb-binlog/maintain.md [hunk 4]
```diff
@@ -112,6 +112,8 @@ Usage of binlogctl:
...
+-show-offline-nodes
+    在用 -cmd pumps 或 -cmd drainers 时使用，这两个命令默认不显示 offline 的节点，仅当明确指定 -show-offline-nodes 时会显示
```
Comment (lilin90) [inline v2.1/reference/tools/tidb-binlog/maintain.md]:
```suggestion
    在用 `-cmd pumps` 或 `-cmd drainers` 命令时使用，这两个命令默认不显示 offline 的节点，仅当明确指定 `-show-offline-nodes` 时会显示
```

File: v3.0/reference/tools/tidb-binlog/maintain.md [hunk 5]
```diff
@@ -112,6 +112,8 @@ Usage of binlogctl:
         PEM 格式的 X509 key 文件的路径
 -time-zone string
     如果设置时区，在 "generate_meta" 模式下会打印出获取到的 tso 对应的时间。例如"Asia/Shanghai" 为 CST 时区，"Local" 为本地时区
+-show-offline-nodes
+    在用 -cmd pumps 或 -cmd drainers 时使用，这两个命令默认不显示 offline 的节点，仅当明确指定 -show-offline-nodes 时会显示
```
Comment (lilin90) [inline v3.0/reference/tools/tidb-binlog/maintain.md]:
```suggestion
    在用 `-cmd pumps` 或 `-cmd drainers` 命令时使用，这两个命令默认不显示 offline 的节点，仅当明确指定 `-show-offline-nodes` 时会显示。
```

File: v3.0/reference/tools/tidb-binlog/maintain.md [hunk 6]
```diff
@@ -112,6 +112,8 @@ Usage of binlogctl:
         PEM 格式的 X509 key 文件的路径
 -time-zone string
     如果设置时区，在 "generate_meta" 模式下会打印出获取到的 tso 对应的时间。例如"Asia/Shanghai" 为 CST 时区，"Local" 为本地时区
```
Comment (lilin90) [inline v3.0/reference/tools/tidb-binlog/maintain.md]:
```suggestion
    如果设置时区，在 "generate_meta" 模式下会打印出获取到的 tso 对应的时间。例如 "Asia/Shanghai" 为 CST 时区，"Local" 为本地时区
```

File: dev/reference/tools/tidb-binlog/maintain.md (hunk 2 above)
Comment (lilin90) [reply dev/reference/tools/tidb-binlog/maintain.md]: 具体如何明确指定呢？

Comment (suzaku) [reply dev/reference/tools/tidb-binlog/maintain.md]: 加上 `-show-offline-nodes` 就行

**Pull request merged**